src/SprykerEco/Zed/Heidelpay/Business/Payment/Transaction/TransactionLogReader.php

```diff
@@ -87,6 +87,70 @@ public function findOrderAuthorizeTransactionLogByIdSalesOrder(int $idSalesOrder
         return $this->buildTransactionTransfer($spyTransactionLog);
     }
 
+    /**
+     * @param int $idSalesOrder
+     *
+     * @return \Generated\Shared\Transfer\HeidelpayTransactionLogTransfer|null
+     */
+    public function findOrderAuthorizeOnRegistrationTransactionLogByIdSalesOrder($idSalesOrder)
+    {
+        $spyTransactionLog = $this->findOrderAuthorizeOnRegistrationTransactionEntity($idSalesOrder);
+
+        if ($spyTransactionLog === null) {
+            return null;
+        }
+
+        return $this->buildTransactionTransfer($spyTransactionLog);
+    }
+
+    /**
+     * @param int $idSalesOrder
+     *
+     * @return \Generated\Shared\Transfer\HeidelpayTransactionLogTransfer|null
+     */
+    public function findQuoteInitializeTransactionLogByIdSalesOrder($idSalesOrder)
+    {
+        $spyTransactionLog = $this->findQuoteInitializeTransactionEntity($idSalesOrder);
```

Comment:
Please fix this file. This should not be that way.

Reply:
Skipped (@alex-galych ).